pages/projects/favourite.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import commonStyles from "../../styles/common.module.scss";
+import styles from "./FrontPage.module.scss";
+import Head from "next/head";
+import { Layouts } from "../../layouts/LayoutWrapper";
+import SideNavBar from "components/SideNavBar";
+import FrontPageBody from "components/FrontPageBody";
+import { useQuery } from "react-query";
+import { getProjects } from "../../services/projectApi";
+import { useRouter } from "next/router";
+import { Typography } from "@equinor/eds-core-react";
+import { SortSelect } from "../../components/SortSelect";
+
+export default function FavoriteProjects(): JSX.Element {
+  const [page, setPage] = useState(1);
+  const itemsPerPage = 15; //Todo: Display as many cards we can fit while still making space for the pagination
```

Comment:
I think I want to fix this one before we release this...
Lets how many to display on a given window.